code/game/gamemodes/miniantags/abduction/machinery/pad.dm

```diff
@@ -47,8 +47,8 @@
 	icon_state = "teleport"
 	duration = 80
 
-/obj/effect/overlay/temp/teleport_abductor/New()
+/obj/effect/overlay/temp/teleport_abductor/Initialize()
+	. = ..()
 	var/datum/effect_system/spark_spread/S = new
 	S.set_up(10,0,loc)
-	S.start()
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?